package.json

```diff
@@ -6,19 +6,27 @@
   "directories": {
```

Comment:
because it is common to multiple projects. Also avoids having node_modules in each projects that uses node